themes/theme-without-style-book-theme/src/css/variables/_button.scss

```diff
@@ -0,0 +1,53 @@
+$btn: () !default;
```

Comment:
You are setting `border-radius: 0` in three places, maybe it would be better to set the global and set rounded border only where needed.

There are three global variables for `border-radius`:

```
$border-radius: 0;
$border-radius-lg: 0;
$border-radius-sm: 0; 
```